index.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="./assets/css/home.css">
+    <script src="https://kit.fontawesome.com/1f634f9b31.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap" rel="stylesheet">
+</head>
+<body>
+    <section class="set-background">
+    <header class="header">
+        <nav class="social-media" aria-label="social media icons">
+            <ul class="social-media-ul">
+                <li><a href=""><img src="./assets/images/social-media-images/facebook.png" alt=""></a></li>
+                <li><a href=""><img src="./assets/images/social-media-images/twitter.png" alt=""></a></li>
+                <li><a href="#">English</a></li>
+                <li><a href="https://github.com/yusufyusice">My github page</a></li>
+                <li><a href="#">Logout</a></li>
+            </ul>
```

Comment:
- [ ] If you have decided to make your navbar fixed please also make sure that the rest of the content moves under it instead of over it by making use of `z-index`

<img width="1501" alt="Screenshot 2023-06-30 at 9 12 30 AM" src="https://github.com/yusufyusie/Africa-Day-2023-Concert/assets/44798044/96d2da84-3bdf-4b85-ac9a-68dbc145d013">